src/components/Footer/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Link } from 'react-router';
+import s from './Footer.module.css';
+
+const Footer = () => {
+  return (
+    <div>
+      <footer className={s.footerContainer}>
+        <div className={s.logo}> 2018 Technology Inc. </div>
+        <div className={s.link}>
```

Comment:
That one is not just a link, link good name for a single child, but not a container of several links

Reply:
better to use `linksContainer` or something like that